test/integration/kubernetes_deploy_test.rb

```diff
@@ -890,4 +890,49 @@ def test_cronjobs_can_be_deployed
     cronjobs = FixtureSetAssertions::CronJobs.new(@namespace)
     cronjobs.assert_cronjob_present("my-cronjob")
   end
+
+  def test_resource_watcher_reports_failed_after_timeout
+    result = deploy_fixtures(
+      "invalid",
+      subset: ["bad_probe.yml", "cannot_run.yml", "missing_volumes.yml"],
+      max_watch_seconds: 15
```

Comment:
This test is flakey, cannot_run.yml, sometimes hits the global timeout instead of failing. I could bump this, but open to other ideas.